shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -23,8 +23,8 @@
             <description>When enabled, the motors will spin at the ChannelNeutral command when armed with zero throttle.</description>
         </field>
 
-        <field name="MotorInputOutputCurveFit" units="-" type="float" elementnames="A,B" defaultvalue="1,1">
-            <description>Actuator mapping of input in [-1,1] to output on [-1,1], using power equation of type a*x^b.</description>
+        <field name="MotorInputOutputCurveFit" units="-" type="float" elements="1" defaultvalue="1">
```

Comment:
> The data seems to show that the "meat" of the curve, i.e. the center point where the propeller spends most of its time, is a better fit with two parameters.

Whatever output setting results in a hover does not change by multiplication.

Multiplying a function by something does not make the corresponding "output" part of the function more linear.

The shape of the curve where hover occurs (or a given speed of forward flight) does not change by virtue of multiplication.